### PR TITLE
Add transitional updater

### PR DESCRIPTION
**Pull request by mdrohmann**
https://www.pivotaltracker.com/story/show/177827563

This certainly feels and looks hacky, but I am fairly certain that is the nature of this story...

"Follow-up" story is here: https://www.pivotaltracker.com/story/show/177974542

### CODE REVIEW

File: activestate.generators.yaml [hunk 1]
```diff
@@ -56,9 +56,13 @@ scripts:
     description: Generate update files
     value: |
       set -e
+      export GOARCH=${1:-amd64}
       $constants.SET_ENV
 
-      export GOARCH=${1:-amd64}
+      # Generate legacy update
+      go build -o $BUILD_TARGET_DIR/$constants.BUILD_TRANSITION_TARGET ./cmd/state-transition-update
```
Comment (Naatan) [inline activestate.generators.yaml:63]: I think we'll need to sign this on Windows, otherwise we might get the UAC prompt?

**Review by Naatan — CHANGES_REQUESTED** [review]
Looks ok to me! Just need to ensure that the executable is signed on Windows.

File: activestate.generators.yaml (hunk 1 above)
Comment (mdrohmann) [reply activestate.generators.yaml:63]: Good point!  I think I'll cramp that in there too, as we are deleting that code soon anyways...

Comment (mdrohmann) [reply activestate.generators.yaml:63]: Shoot. That won't work, because signing is only done on CI.  Okay, then I have to split it up.

**Review by Naatan — APPROVED** [review]

**Pull request merged**